app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -11,7 +11,7 @@
 <%= presenter.attribute_to_html(:identifier, render_as: :linked, search_field: 'identifier_tesim') %>
 <%= presenter.attribute_to_html(:keyword, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:date_created, render_as: :linked, search_field: 'date_created_desim') %>
-<%= presenter.attribute_to_html(:date) %>
+<%= presenter.attribute_to_html(:date, label: "EDTF Date") %>
```

Comment:
Over in epigaea, @no-reply and @mark-dce requested that we don't use labels in the views, but use the locale file instead. 

You'll need to change it in the blacklight and simple form sections of the locale file:
https://github.com/curationexperts/epigaea/blob/master/config/locales/hyrax.en.yml

Reply:
Ah okay, thanks @little9.